src/qibocal/protocols/characterization/flipping_signal.py

```diff
@@ -51,12 +55,39 @@ class FlippingSignalData(Data):
 
     resonator_type: str
     """Resonator type."""
+    detuning: float
```

Comment:
```suggestion
    detuning: Optional[float]
```
It can be `None` in FlippingSignalData too, right?

Reply:
Uhm, not really if it is not provided by the parameters it is set to 0 and it is then passed to data.
Maybe I should remove also the optional from parameters at this point since it cannot be None

Reply:
Ok, it makes sense then.